nvidia_jetson/assets/dashboards/nvidia_jetson.json

```diff
@@ -0,0 +1,473 @@
+                    },
+                    {
+                        "aggregator": "avg",
+                        "alias": "Largest free block",
+                        "cell_display_mode": [
+                            "number"
+                        ],
+                        "q": "top(avg:nvidia.jetson.mem.lfb{$host} by {host}, 10, 'mean', 'desc')"
+                    }
+                ],
+                "title": "Largest free memory block per host",
+                "type": "query_table"
+            },
+            "id": 743229523155656,
+            "layout": {
+                "height": 2,
+                "width": 4,
+                "x": 0,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "area",
+                        "on_right_yaxis": false,
+                        "q": "avg:nvidia.jetson.iram.total{$host} by {host}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "purple"
+                        }
+                    },
+                    {
+                        "display_type": "area",
+                        "on_right_yaxis": false,
+                        "q": "avg:nvidia.jetson.iram.used{$host} by {host}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "purple"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "title": "IRAM Usage",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 1617838839011792,
+            "layout": {
+                "height": 2,
+                "width": 4,
+                "x": 4,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "autoscale": true,
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nvidia.jetson.iram.lfb{$host}"
+                    }
+                ],
+                "title": "Largest IRAM free block",
+                "type": "query_value"
+            },
+            "id": 2743306043181106,
+            "layout": {
+                "height": 1,
+                "width": 1,
+                "x": 8,
+                "y": 6
+            }
+        }
+    ]
+}
```

Comment:
can we delete the other dashboard JSON file `/dashboards/nvidia_jetson_overview.json`?